proto/storage/cluster.proto

```diff
@@ -94,6 +95,43 @@ message AdmissionControllerConfig {
   bool enforce_on_updates = 5;
 }
 
+message ProcessConfig {
+  bool enabled = 1;
+}
+
+message NetworkConnectionConfig {
+  bool enabled = 1;
+  bool aggregate_external = 2;
+}
+
+message NetworkEndpointConfig {
```

Comment:
Your PR defines `processes_listening_on_port` and `networkEndpointStatus` but I can't find it in the proto definitions. Am I missing something here?

Reply:
I don't see where my PR defines `processes_listening_on_port` or `networkEndpointStatus`. I did have those in earlier commits, but changed the names, based on feedback from @robbycochran

Reply:
Okay, I think I looked at outdate code in this case.